gap/zmq.g

```diff
@@ -10,10 +10,10 @@
 ##
 
 
-BindGlobal("SynchronizationFamily",
-        NewFamily("SynchronizationFamily", IsObject));
+BindGlobal("ZmqSynchronizationFamily",
+        NewFamily("ZmqSynchronizationFamily", IsObject));
 DeclareFilter("IsZmqSocket", IsObject and IsInternalRep);
-BindGlobal("TYPE_ZMQ_SOCKET", NewType(SynchronizationFamily, IsZmqSocket));
+BindGlobal("TYPE_ZMQ_SOCKET", NewType(ZmqSynchronizationFamily, IsZmqSocket));
```

Comment:
Hmm, I wonder why it was using `SynchronizationFamily`? Probably because it used that family while this code was in the library (but why did it do that? Just laziness, because that family was conveniently available, or is there a deeper reason? Maybe @rbehrends remembers)...

Anyway, I think the name `ZmqSynchronizationFamily` doesn't make much sense. We also don't need to store the family, as we only use it one. So we could just do this instead, and delete the family above:
```suggestion
BindGlobal("TYPE_ZMQ_SOCKET", NewType(NewFamily("ZmqFamily", IsObject), IsZmqSocket));
```

Reply:
Indeed... one more change needed then.